2025-09-06 19:24:32,191 - Base/baseSendNginx.py - 71 - INFO - allure 报告存在准备发送到nginx: D:\github_program\TestFramework_Po\Report\ALLURE\Report
2025-09-06 19:24:32,191 - Base/baseSendNginx.py - 48 - INFO - common: scp -v -r -P 22 -o ConnectTimeout=10 -o PreferredAuthentications=publickey -o ServerAliveInterval=30 -o ServerAliveCountMax=3 D:\github_program\TestFramework_Po\Report\ALLURE\Report root@113.44.217.121:/usr/share/nginx/html/report/allure
2025-09-06 19:24:32,743 - Base/baseSendNginx.py - 55 - ERROR - 发送失败: Executing: program ssh.exe host 113.44.217.121, user root, command sftp
OpenSSH_for_Windows_9.5p1, LibreSSL 3.8.2
debug1: Connecting to 113.44.217.121 [113.44.217.121] port 22.
debug1: fd 3 clearing O_NONBLOCK
debug1: Connection established.
debug1: identity file C:\\Users\\MSI/.ssh/id_rsa type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_rsa-cert type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_ecdsa type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_ecdsa-cert type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_ecdsa_sk type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_ecdsa_sk-cert type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_ed25519 type 3
debug1: identity file C:\\Users\\MSI/.ssh/id_ed25519-cert type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_ed25519_sk type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_ed25519_sk-cert type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_xmss type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_xmss-cert type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_dsa type -1
debug1: identity file C:\\Users\\MSI/.ssh/id_dsa-cert type -1
debug1: Local version string SSH-2.0-OpenSSH_for_Windows_9.5
debug1: Remote protocol version 2.0, remote software version OpenSSH_8.9p1 Ubuntu-3ubuntu0.13
debug1: compat_banner: match: OpenSSH_8.9p1 Ubuntu-3ubuntu0.13 pat OpenSSH* compat 0x04000000
debug1: Authenticating to 113.44.217.121:22 as 'root'
debug1: load_hostkeys: fopen C:\\Users\\MSI/.ssh/known_hosts2: No such file or directory
debug1: load_hostkeys: fopen __PROGRAMDATA__\\ssh/ssh_known_hosts: No such file or directory
debug1: load_hostkeys: fopen __PROGRAMDATA__\\ssh/ssh_known_hosts2: No such file or directory
debug1: SSH2_MSG_KEXINIT sent
debug1: SSH2_MSG_KEXINIT received
debug1: kex: algorithm: curve25519-sha256
debug1: kex: host key algorithm: ssh-ed25519
debug1: kex: server->client cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: kex: client->server cipher: chacha20-poly1305@openssh.com MAC: <implicit> compression: none
debug1: expecting SSH2_MSG_KEX_ECDH_REPLY
debug1: SSH2_MSG_KEX_ECDH_REPLY received
debug1: Server host key: ssh-ed25519 SHA256:y9Twys6m8YgO1pPnS1iKaihsdzQB0CQW7J6jglxL9hI
debug1: load_hostkeys: fopen C:\\Users\\MSI/.ssh/known_hosts2: No such file or directory
debug1: load_hostkeys: fopen __PROGRAMDATA__\\ssh/ssh_known_hosts: No such file or directory
debug1: load_hostkeys: fopen __PROGRAMDATA__\\ssh/ssh_known_hosts2: No such file or directory
debug1: Host '113.44.217.121' is known and matches the ED25519 host key.
debug1: Found key in C:\\Users\\MSI/.ssh/known_hosts:2
debug1: ssh_packet_send2_wrapped: resetting send seqnr 3
debug1: rekey out after 134217728 blocks
debug1: SSH2_MSG_NEWKEYS sent
debug1: expecting SSH2_MSG_NEWKEYS
debug1: ssh_packet_read_poll2: resetting read seqnr 3
debug1: SSH2_MSG_NEWKEYS received
debug1: rekey in after 134217728 blocks
debug1: get_agent_identities: ssh_get_authentication_socket: No such file or directory
debug1: Will attempt key: C:\\Users\\MSI/.ssh/id_rsa 
debug1: Will attempt key: C:\\Users\\MSI/.ssh/id_ecdsa 
debug1: Will attempt key: C:\\Users\\MSI/.ssh/id_ecdsa_sk 
debug1: Will attempt key: C:\\Users\\MSI/.ssh/id_ed25519 ED25519 SHA256:GixIrv1ayi8JItbI6LjvwKDf0U8PhJZkYhTn1BDz4I0
debug1: Will attempt key: C:\\Users\\MSI/.ssh/id_ed25519_sk 
debug1: Will attempt key: C:\\Users\\MSI/.ssh/id_xmss 
debug1: Will attempt key: C:\\Users\\MSI/.ssh/id_dsa 
debug1: SSH2_MSG_EXT_INFO received
debug1: kex_input_ext_info: server-sig-algs=<ssh-ed25519,sk-ssh-ed25519@openssh.com,ssh-rsa,rsa-sha2-256,rsa-sha2-512,ssh-dss,ecdsa-sha2-nistp256,ecdsa-sha2-nistp384,ecdsa-sha2-nistp521,sk-ecdsa-sha2-nistp256@openssh.com,webauthn-sk-ecdsa-sha2-nistp256@openssh.com>
debug1: kex_ext_info_check_ver: publickey-hostbound@openssh.com=<0>
debug1: SSH2_MSG_SERVICE_ACCEPT received
debug1: Authentications that can continue: publickey,password
debug1: Next authentication method: publickey
debug1: Trying private key: C:\\Users\\MSI/.ssh/id_rsa
debug1: Trying private key: C:\\Users\\MSI/.ssh/id_ecdsa
debug1: Trying private key: C:\\Users\\MSI/.ssh/id_ecdsa_sk
debug1: Offering public key: C:\\Users\\MSI/.ssh/id_ed25519 ED25519 SHA256:GixIrv1ayi8JItbI6LjvwKDf0U8PhJZkYhTn1BDz4I0
debug1: Authentications that can continue: publickey,password
debug1: Trying private key: C:\\Users\\MSI/.ssh/id_ed25519_sk
debug1: Trying private key: C:\\Users\\MSI/.ssh/id_xmss
debug1: Trying private key: C:\\Users\\MSI/.ssh/id_dsa
debug1: No more authentication methods to try.
root@113.44.217.121: Permission denied (publickey,password).
scp: Connection closed
